worldedit-core/src/main/java/com/sk89q/worldedit/command/util/SuggestionHelper.java

```diff
@@ -88,26 +88,23 @@ public static Stream<String> getBlockPropertySuggestions(String blockType, Strin
                     // only property, no value yet
                     final List<? extends Property<?>> matchingProps = propertyMap.entrySet().stream()
                             .filter(p -> !matchedProperties.contains(p.getKey()) && p.getKey().startsWith(matchProp))
-                            .map(Map.Entry::getValue).collect(Collectors.toList());
-                    switch (matchingProps.size()) {
-                        case 0:
-                            return propertyMap.keySet().stream().filter(p -> !matchedProperties.contains(p)).map(prop ->
-                                    lastValidInput + prop + "=");
-                        case 1:
-                            return matchingProps.get(0).getValues().stream().map(val ->
-                                    lastValidInput +  matchingProps.get(0).getName() + "="
-                                            + val.toString().toLowerCase(Locale.ROOT));
-                        default:
-                            return matchingProps.stream().map(p -> lastValidInput + p.getName() + "=");
-                    }
+                            .map(Map.Entry::getValue).toList();
+                    return switch (matchingProps.size()) {
+                        case 0 -> propertyMap.keySet().stream().filter(p -> !matchedProperties.contains(p)).map(prop ->
+                            lastValidInput + prop + "=");
+                        case 1 -> matchingProps.get(0).getValues().stream().map(val ->
+                            lastValidInput + matchingProps.get(0).getName() + "="
+                                + val.toString().toLowerCase(Locale.ROOT));
```

Comment:
Can you format these lambdas a little better, it's kinda ugly. Maybe:
```suggestion
                        case 0 -> propertyMap.keySet().stream()
                            .filter(p -> !matchedProperties.contains(p))
                            .map(p -> lastValidInput + p + "=");
                        case 1 -> matchingProps.get(0).getValues().stream()
                            .map(val ->
                                lastValidInput + matchingProps.get(0).getName() + "="
                                    + val.toString().toLowerCase(Locale.ROOT)
                            );
```